core/amber/src/main/scalapb/edu/uci/ics/amber/engine/architecture/worker/controlcommands/ControlCommandV2.scala

```diff
@@ -1834,6 +1834,7 @@ object SchedulerTimeSlotEventV2 extends scalapb.GeneratedMessageCompanion[edu.uc
 @SerialVersionUID(0L)
```

Comment:
Those files are generated by scala protobuf. I don't see the files generated by python protobuf?

Reply:
run scripts/python-proto-gen.sh

Reply:
I was able to run scripts/python-proto-gen.sh and no new modifications were added to the python files.